_insights/insights.md

```diff
@@ -0,0 +1,82 @@
+
+But rather than focus on labs, the more pertinent question may be why aren’t the characteristics of lab spaces, not the norm and where they do exist, why are they so scarce? Teams are increasingly being asked to work more collaboratively, creatively and to adopt agile practices but current workspaces don’t mirror the qualities people say they valued in the Service Innovation Lab or similar spaces.
+
+#### How might we...
+
+- Shift the focus from how innovation labs are designed to how we create workspaces that meet the needs of work people are asked to do? Why are they so scarce?
+- Apply the principles and practice of innovation labs into new settings without the need for formal/physical lab infrastructure?
+
+### 4. Joined up working by design
+
+A focus on ‘life events’ and integrated services established a work programme that required collaboration by default. As no single agency was responsible for every aspect of a life event, it needed teams to work together and recognise their interdependence. More importantly, it put the lived experience of people at the centre or the work.
+
+But arranging government around life events with current structures presents challenges, and competing priorities creates fragility. Agency priorities can also override shared priorities. It requires people to look beyond their own agency boundaries, resources and accountabilities.
+
+#### How might we...
+
+- Develop work programmes, processes and structures that foster collaboration and joined-up working by design? What would it take?
+- Create a more influential and more visible role for people/service users in policy development, implementation and delivery?
+
+### 5. The precariousness of funding for new ways of working holds us back
+
+The system-level conditions (resources, ownership, authority) for innovation work are persistently precarious and often short term for projects and labs/innovation teams. While this can be a helpful driver for innovation, in a public sector context innovation work is often grappling with complex long-term issues.
+
+But short term funding cycles often leads to time and effort being diverted towards justifying continued support and funding and away from the work itself. It can create pressures to focus on quick wins and justifying the work, over system-level change and a longer-term focus.
+
+#### How might we…
+
+- Develop approaches to funding and resourcing labs and innovation teams that incentivise long term thinking and outcomes?
+- Better demonstrate value and impact in the short term that supports a long term approach?
+
+(NOTE - here's a couple more to added here too)
```

Comment:
Placeholder alert!

Reply:
:stuck_out_tongue: Haha, good to note! :+1:  Thanks @mermop